File: src/_7kyu/_7kyu_recursion_number_1_factorial/FactorialTest.js
describe(`Basic tests`, _ => {
    it(`factorial(0)`, _ => {
        Test.assertEquals(factorial(0), 1);
    });
    it(`factorial(1)`, _ => {
        Test.assertEquals(factorial(1), 1);
    });
    it(`factorial(2)`, _ => {
        Test.assertEquals(factorial(2), 2);
    });
    it(`factorial(3)`, _ => {
        Test.assertEquals(factorial(3), 6);
    });
});

File: src/_7kyu/_7kyu_recursion_number_1_factorial/FactorialTest.cs
namespace Solution {
	using NUnit.Framework;
	using System;

	[TestFixture]
	public class RecursionTests { [Test]
		public void BasicTests() {
			Assert.AreEqual(1, Recursion.Factorial(0));
			Assert.AreEqual(1, Recursion.Factorial(1));
			Assert.AreEqual(2, Recursion.Factorial(2));
			Assert.AreEqual(6, Recursion.Factorial(3));
		}
	}
}